yarn.lock

```diff
@@ -5290,9 +5290,9 @@ kind-of@^6.0.0, kind-of@^6.0.2:
   resolved "https://registry.verdaccio.org/kind-of/-/kind-of-6.0.2.tgz#01146b36a6218e64e58f3a8d66de5d7fc6f6d051"
   integrity sha512-s5kLOcnH0XqDO+FvuaLX8DDjZ18CGFk7VygH40QoKPUQhW4e2rvM0rwUq0t8IQDOwYSeLK01U90OjzBTme2QqA==
 
-kleur@^3.0.2:
+kleur@3.0.3, kleur@^3.0.2:
   version "3.0.3"
-  resolved "https://registry.verdaccio.org/kleur/-/kleur-3.0.3.tgz#a79c9ecc86ee1ce3fa6206d1216c501f147fc07e"
+  resolved "https://registry.npmjs.org/kleur/-/kleur-3.0.3.tgz#a79c9ecc86ee1ce3fa6206d1216c501f147fc07e"
```

Comment:
@juanpicado is the `.npmrc` file in the repo correct? It uses the npm registry.

Reply:
:-) my bad ... I use `nrm` command often.

Reply:
ahh, well, the lock file points to our registry in purpose. So, we do not have to deal with registry timeouts.